zproject/dev_settings.py

```diff
@@ -111,7 +111,22 @@
 
 if FAKE_LDAP_MODE:
     import ldap
-    from django_auth_ldap.config import LDAPSearch
+    from django_auth_ldap.config import (
+        LDAPSearch, LDAPSearchUnion, LDAPGroupQuery, PosixGroupType,
+        MemberDNGroupType, NestedMemberDNGroupType,
+        GroupOfNamesType, NestedGroupOfNamesType,
+        GroupOfUniqueNamesType, NestedGroupOfUniqueNamesType,
+        ActiveDirectoryGroupType, NestedActiveDirectoryGroupType,
+        OrganizationalRoleGroupType, NestedOrganizationalRoleGroupType
+    )
+    # Suppress pyflakes warnings about unused modules.
+    assert (LDAPSearchUnion and LDAPGroupQuery and PosixGroupType and
+            MemberDNGroupType and NestedMemberDNGroupType and
+            GroupOfNamesType and NestedGroupOfNamesType and
+            GroupOfUniqueNamesType and NestedGroupOfUniqueNamesType and
+            ActiveDirectoryGroupType and NestedActiveDirectoryGroupType and
+            OrganizationalRoleGroupType and NestedOrganizationalRoleGroupType)
```

Comment:
I don't think we need these in `dev_settings.py`?

Reply:
I don't insist, I just thought that these classes can be useful during development too. I'll remove them.